wwwroot/js/site.js

```diff
@@ -14,7 +14,7 @@ window.addEventListener('DOMContentLoaded', event => {
       sidebarToggle.addEventListener('click', event => {
           event.preventDefault();
           document.body.classList.toggle('sb-sidenav-toggled');
-          localStorage.setItem('sb|sidebar-toggle', document.body.classList.contains('sb-sidenav-toggled'));
+          // localStorage.setItem('sb|sidebar-toggle', document.body.classList.contains('sb-sidenav-toggled'));
```

Comment:
Is this disabled for now?

Reply:
Yeah, I want to think about this one a bit more before getting rid of it completely, but I think it will get removed eventually. I'll file an item to figure that out.